js/bookmarkConverter.js

```diff
@@ -81,6 +84,33 @@ const bookmarkConverter = {
       })
         .catch(reject)
     })
+  },
+  initialize: function () {
+    // how often to create a new backup file
+    const interval = (3 * 24 * 60 * 60 * 1000)
```

Comment:
Could this interval perhaps be made a user setting?

It would add some complexity. At the same time the frequency of adding bookmarks probably differs a bunch from user to user.